addons/more-links/addon.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "More links",
+  "description": "Adds links for URLs pointing outside scratch.mit.edu.",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": [
+        "https://scratch.mit.edu/projects/*",
+        "https://scratch.mit.edu/studios/*",
+        "https://scratch.mit.edu/users/*"
+      ]
+    }
+  ],
+  "tags": ["community"],
+  "enabled_by_default": true
```

Comment:
This should not be enabled by default.

Reply:
Eh - debatable. But I agree, I don't think it's a good idea to enable so many addons to previous Scratch Messaging Extension users, specially considering that extension currently makes no changes to the Scratch website.